docs/user-guide/progressive-release/extending.md

```diff
@@ -0,0 +1,122 @@
+---
+template: main.html
+---
+
+# Extending progressive release to other deployment environments
+
+The `release` chart can be easily extended to include other deployment environments. Please consider contributing any extensions to Iter8. We briefly describe how to extend the chart for an [Knative](https://knative.dev/docs/) application. 
+
+## Approach
+
+Modify the release chart after forking the [Iter8 project](https://github.com/iter8-tools/iter8). The chart to be extended is in the `charts/release` sub-folder. The file `release.yaml` is the starting point. For each valid environment, the chart contains a set of files defining the resources that should be created.  These may include:
```

Comment:
```suggestion
Modify the release chart after forking the [Iter8 project](https://github.com/iter8-tools/iter8). The chart to be extended is in the `charts/release` sub-folder. The file `release.yaml` is the starting point. For each valid environment, the chart contains a set of files defining the resources that should be created. These may include:
```

Reply:
I wonder if it's necessary to mention forking the Iter8 project. I think this passage could probably be shortened.
```suggestion
The progressive release chart can be found in the `charts/release` sub-folder of the Iter8 GitHub repository. The file `release.yaml` is the starting point. For each valid environment, the chart contains a set of files defining the resources that should be created.  These may include:
```